agent/structs/structs.go

```diff
@@ -72,6 +72,46 @@ const (
 	SystemMetadataRequestType                   = 31
 )
 
+// if a new request type is added above it must be
+// added to the map below
+
+// requestTypeStrings is used for snapshot enhance
+// any new request types added must be placed here
+var requestTypeStrings = map[MessageType]string{
+	0:  "Register",
```

Comment:
Would it be good to use the request type constance from a few lines before as indexes into this list.

If we ever were refactoring those constants it would be easier to find them in this map that way.